### PR TITLE
TASK: Remove contructors from react-ui-components

### PR DESCRIPTION
**Pull request by kitsunet**
This removes constructors from all components in the
react-ui-components package and uses arrow functions
as class properties instead of binding. This makes
the code a bit more concise.

Related: #979

### CODE REVIEW

**Comment by kitsunet** [comment]
Note: I was very liberal with removing. There were two places were props were put in state in the constructor. I removed that because I felt in both cases it was not really needed, but make sure you check.

**Comment by kitsunet** [comment]
Apparently I broke some tests... Will look into it. Might need to bring one or two constructors back...

**Comment by Inkdpixels** [comment]
@kitsunet Could I support you in this PR somehow?

**Comment by kitsunet** [comment]
I just forgot 😆 Feel free to take over, I am right now entangled in something else... Can do later htough.

**Comment by Inkdpixels** [comment]
@kitsunet alrighty, will take over, thanks a bunch :) :heart:

File: packages/neos-ui-guest-frame/src/InlineUI/NodeToolbar/index.js [hunk 1]
```diff
@@ -37,10 +37,7 @@ export default class NodeToolbar extends PureComponent {
         isSticky: false
     };
 
-    constructor() {
-        super();
-        this.iframeWindow = document.getElementsByName('neos-content-main')[0].contentWindow;
-    }
+    iframeWindow = document.getElementsByName('neos-content-main')[0].contentWindow;
```
Comment (grebaldi) [inline packages/neos-ui-guest-frame/src/InlineUI/NodeToolbar/index.js]: Maybe we can use this opportunity to replace this call with the helper method from `@neos-project/neos-ui-guest-frame/src/dom`

**Pull request merged**